TCMSApp/src/client/app/tests/tests-create.controller.js

```diff
@@ -0,0 +1,36 @@
+(function () {
+    'use strict';
+
+    angular
+        .module('app.tests')
+        .controller('TestsCreateController', TestsCreateController);
+
+    TestsCreateController.$inject = ['logger', 'FakeTestsFactory', '$uibModal', 'TestsService'];
+    /* @ngInject */
+    function TestsCreateController(logger, FakeTestsFactory, $uibModal, TestsService) {
+        var vm = this;
+
+        activate();
+
+        vm.stepList = [1,2];
+        vm.addStep = addStep;
+        vm.delStep = delStep;
+        vm.currentSuite = TestsService.getCurrentSuite();
+
+        function activate() {
+            logger.info('Activated New Case View');
+        }
+
+        function addStep(){
+            var stepCount = vm.stepList.length;
+            vm.stepList.push(stepCount + 1);
```

Comment:
we will make the separate task for actually pushing real test steps, not numbers?